### ISSUE TITLE
CSS Modules Not Working

### ISSUE THREAD

**Issue by itwasmattgregg** [timhagn/gatsby-background-image#7]
I'm using CSS modules like so:
```
<BackgroundImage Tag="div" fluid={background} className={styles.blog_featureSections_block} backgroundColor={`#040e18`}>
  <h2>
    <Link to={`/category/dev`}>Dev</Link>
  </h2>
</BackgroundImage>
```
and I'm getting errors about the className not being defined. I get the same error when I omit the className altogether which I feel like I should be able to do. There is nothing in the documentation saying that className is required.

Here's my stack trace:
<img width="1024" alt="screen shot 2019-02-25 at 3 04 15 pm" src="https://user-images.githubusercontent.com/7470936/53368661-f4bf1680-390e-11e9-8ca9-18b6cf2e3dc8.png">

**Comment by itwasmattgregg** [comment]
Realized that I had removed all css rules from that class in my stylesheet so it really was undefined. Oops. Maybe would be worth adding to documentation that a class with some sort of styles are required for this component to work.

**Comment by timhagn** [comment]
Though you closed this issue yourself, @itwasmattgregg, I just took a look on your second suggestion.
From your stack trace for `BackgroundUtils.js`, might I which version you are using?
Cause your last mentioned issue arose beforehand (see #5) and behavior was changed  and more if-clauses added in `0.2.3` (current at the time of your issue, `0.2.31` right now).
Should anymore issues arise, I'm happy to help : ).
